eth_tester/utils/backend_testing.py

```diff
@@ -268,11 +268,28 @@ def test_send_raw_transaction_invalid_raw_transaction(self, eth_tester):
         with pytest.raises(rlp.exceptions.DecodingError):
             eth_tester.send_raw_transaction(invalid_transaction_hex)
 
-    def test_send_transaction(self, eth_tester):
+    @pytest.mark.parametrize(
+        'test_transaction',
+        (
+            SIMPLE_TRANSACTION,
+            {
+                "to": BURN_ADDRESS,
+                "gas_price": 1,
+                "value": 0,
+                "gas": 21000,
+                "nonce": 1,
+            },
```

Comment:
Can you replace this with a constant at the top of the module next to where `SIMPLE_TRANSACTION` is defined.

```python
from cytoolz import assoc

TRANSACTION_WITH_NONCE = assoc(SIMPLE_TRANSACTION, 'nonce', 1)
```

Reply:
👍